src/main/java/org/cactoos/text/CapitalizedText.java

```diff
@@ -53,15 +53,15 @@ public CapitalizedText(final Text text) {
     @Override
     public String asString() throws IOException {
         return new JoinedText(
-            new TransformedIterable<String, Text>(
+            new TransformedIterable<Text, Text>(
                 new SplitText(this.origin),
-                new Func<String, Text>() {
+                new Func<Text, Text>() {
                     @Override
-                    public Text apply(final String input) throws Exception {
+                    public Text apply(final Text input) throws Exception {
```

Comment:
@Timmeey Primary one constructor as private? With all due to respect, but no way! 😄 In general, I don't like any code in constructors. I know is ok, because it's not executing, just creating a new object, but I think it not elegant. So, we have these options:
1. To declare as a public class, as it's now;
2. To declare the function as anonymous, as it was before;
3. To declare in the secundary constructor, as I suggested;
4. To declare in the primary constructor, as you suggested;
5. To declare as a private class.

So, what do you prefer?
@yegor256 Would do you like give us your opinion?